src/test/java/org/mastodon/mamut/fitting/FitEllipsoidPluginDemo.java

```diff
@@ -0,0 +1,62 @@
+/*-
```

Comment:
Is there some rule in the Mastodon repos that I am not aware of that licensing info is added to every single class?

Reply:
Yes, there is. The release-version script that we use for cutting releases even enforces this rule. You can also use maven to add the version headers `mvn license:update-file-headers`

Reply:
Okay please resolve.